cachet_url_monitor/webhook.py

```diff
@@ -0,0 +1,26 @@
+from typing import Dict, Optional
+
+import requests
+
+
+class Webhook:
+    url: str
+    params: Dict[str, str]
+
+    def __init__(self, url: str, params: Dict[str, str]):
+        self.url = url
+        self.params = params
+
+    def push_incident(self, title: str, message: str):
+        format_args = {
```

Comment:
What do you think about also sharing the component id?

Reply:
I didn't find any use for it here. There are many variables which might be useful to include for the interpolation, but I'd do that once somebody needs them.

Reply:
Sounds good to me.